tools/building.py

```diff
@@ -1248,6 +1246,13 @@ def emit_debug_on_side(wasm_file):
   shutil.move(wasm_file, wasm_file_with_dwarf)
   strip(wasm_file_with_dwarf, wasm_file, debug=True)
 
+  # Strip code and data from the debug file to limit its size. The other known
+  # sections are still required to correctly interpret the DWARF info.
+  # TODO(dschuff): Also strip the DATA section? To make this work we'd need to
+  # either allow "invalid" data segment name entries, or maybe convert the DATA
+  # to a DATACOUNT section.
+  strip(wasm_file_with_dwarf, wasm_file_with_dwarf, sections=['CODE'])
```

Comment:
Why is `producers` in lowercase but `CODE` in upper?

Reply:
CODE is a known section, and known sections are displayed and matched by LLVM tools with capitalized names (whereas all the recognized custom section names happen to be lowercase)